### ISSUE TITLE
[Suggestion] Anti-Doorbreach Module

### ISSUE THREAD

**Issue by Delfite** [chuxiaaaa/AntiCheat#9]
There is a mod called [Doorbreach](https://thunderstore.io/c/lethal-company/p/Nyxchrono/DoorBreach/) that allows the client to breach doors with any melee weapon. I recommend adding a toggle-able module to detect if a player opens a locked door without a key or lockpicker and kick them if they meet the requirements to do so.

**Comment by chuxiaaaa** [comment]
I'm not sure if it's necessary. Everyone can discuss it under this issues.

**Comment by CoolLKKPS** [comment]
TL;DR
ok sure, but make this feature are disabled by default, but also i sug disable detect mini-map mod feature by default

for vanilla anti-cheat, its necessary, but for gameplay like modpack experience or something else, the answer is don't, so in my opinion is 70 30, depends on who hosting the game
just like mini-map mod, this feature can be easily add in this mod i believe, just need to check who are using this mod, but for mod community, unless this can confirm its a 100% cheat mod on many normal player side, otherwise that will hurt some people even they not playing vanilla completely, bc in the end, those all depends on community standard not official UA, the host has the right to control the gameplay, u can choose to join or not, fair enough

**Comment by CoolLKKPS** [comment]
\* in english translation, i explained why mini-map is unfair, which that mod can show u where has loot where is danger, yes in vanilla its cheating bc the config file doesn't need to sync with the host, but for modpack that's not really necessary

\* 在英语翻译中，我解释了为什么小地图是不公平的，这个模组可以显示哪里有敌人以及哪里危险，是的对于原版这是作弊，因为它的配置文件不需要与主机同步就能用，但对于整合包就不一定
